src/components/Calendar/src/Calendar.vue

```diff
@@ -0,0 +1,460 @@
+<template>
+	<div
+		:class="$s.Calendar"
+		v-bind="$attrs"
+		v-on="$listeners"
+	>
+		<div :class="$s.CalendarHeader">
+			<m-button
+				:disabled="shouldDisableHeaderButton(-1)"
+				:color="headerButtonColor"
+				size="small"
+				variant="primary"
+				tabindex="-1"
+				@click="incMonth(-1)"
+			>
+				<chevron-left
+					:class="$s.CalendarHeaderButtonIcon"
+				/>
+			</m-button>
+
+			<span :class="$s.CalendarHeaderTitle">
+				{{ monthName }}
+			</span>
+
+			<m-button
+				:disabled="shouldDisableHeaderButton(1)"
+				size="small"
+				variant="primary"
+				tabindex="-1"
+				color="#f2f2f2"
+				@click="incMonth(1)"
+			>
+				<chevron-right
+					:class="$s.CalendarHeaderButtonIcon"
+				/>
+			</m-button>
+		</div>
+		<table :class="$s.CalendarTable">
+			<thead>
+				<tr>
+					<th
+						v-for="day in weekdays"
+						:key="`day-${day}`"
+						:class="$s.DateHeaderCell"
+					>
+						{{ day }}
+					</th>
+				</tr>
+			</thead>
+			<tbody>
+				<tr
+					v-for="(week, wi) in weeks"
+					:key="`week-${wi}`"
+				>
+					<td
+						v-for="(date, di) in week"
+						:key="`date-${wi-di}`"
+						:class="$s.DateCell"
+						@click.prevent="emitDate(date)"
+					>
+						<button
+							v-if="date"
+							:class="[$s.DateCellButton, {
+								[$s.selected]: isDateSelected(date),
+								[$s.disabled]: isDateDisabled(date),
+								[$s.today]: isToday(date)
+							}]"
+							type="button"
+							tabindex="-1"
+						>
+							{{ date }}
+						</button>
+					</td>
+				</tr>
+			</tbody>
+		</table>
+	</div>
+</template>
+
+<script>
+import {
+	startOfWeek,
+	startOfMonth,
+	eachDayOfInterval,
+	getDay,
+	lastDayOfMonth,
+	addDays,
+	format,
+	isToday,
+	isAfter,
+	isSameDay,
+	isSameMonth,
+	isSameYear,
+	addMonths,
+	parse,
+} from 'date-fns';
+import { MButton } from '@square/maker/components/Button';
+import ChevronLeft from '@square/maker-icons/ChevronLeft';
+import ChevronRight from '@square/maker-icons/ChevronRight';
+
+const DATE_STRING_FORMAT = 'yyyy-MM-dd';
+const isDateFormat = (string) => !string || /^\d{4}-\d{2}-\d{2}$/.test(string); // YYYY-MM-DD
+const getLocale = (locale) => import(`date-fns/locale/${locale}/index.js`);
```

Comment:
JS libraries can't have asynchronous imports because that's an application concern.

Let's remove the date-fns localization dependency and use the native browser localization like it was doing before.

Use date-fns for basic date hadnling though.